CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
```

Comment:
Would you please move this change log under `packages/spotlight`?
We don't change this file while developing. It will properly be changed while we are releasing.